### DISCUSSION TITLE
Nest to integrate Mercurius as a standard dependency for their GraphQL module

### DISCUSSION THREAD

**Discussion by smolinari** [mercurius-js/mercurius#713] [General]
Hi,

I just wanted to let everyone in this community know, the NestJS makers are adding Mercurius (as the Fastify solution) next to Apollo Server (as the Express solution) for their GraphQL module. So, I'd imagine through that exposure, this community will be growing in leaps and bounds. :) I know I'm happy this is happening. 

Scott

**Comment by mcollina** [comment]
That's awesome! Thanks for letting us know!

**Comment by smolinari** [comment]
FYI, Nestjs just released it's version 10, which includes Mercurius integration. 

![image](https://user-images.githubusercontent.com/3314957/153709694-02fddc4e-8917-4efd-987a-3a05c8df21c1.png)

Scott
